mitodl/mit-open
# two separate search inputs

### What are the relevant tickets?
closes https://github.com/mitodl/hq/issues/4632

### Description (What does it do?)
Prior to this PR the search inputs on the front page and search page was the same component. https://github.com/mitodl/mit-open/pull/1104 updated the input. However we do not want to update the input on the search page so we need two separate components

frontends/mit-open/src/page-components/SearchDisplay/SearchInput.tsx is mostly the latest version of frontends/ol-components/src/components/SearchInput/SearchInput.tsx

and 

frontends/mit-open/src/pages/HomePage/SearchInput.tsx is mostly the version frontends/ol-components/src/components/SearchInput/SearchInput.tsx before my latest pr

### Screenshots (if appropriate):
<img width="1655" alt="Screenshot 2024-06-17 at 9 49 23 PM" src="https://github.com/mitodl/mit-open/assets/1934992/b39d498e-4f6e-4ee7-8489-caa07e0d84e0">

<img width="379" alt="Screenshot 2024-06-17 at 9 49 44 PM" src="https://github.com/mitodl/mit-open/assets/1934992/4ee76fc2-25bf-41d3-9465-8443399cf1a1">

<img width="1661" alt="Screenshot 2024-06-17 at 9 48 40 PM" src="https://github.com/mitodl/mit-open/assets/1934992/4c7c8f2c-587a-441f-9d84-5ff43944718b">

<img width="377" alt="Screenshot 2024-06-17 at 9 48 59 PM" src="https://github.com/mitodl/mit-open/assets/1934992/e23e0d45-a564-4e65-9641-5149b055769c">

### How can this be tested?
Go to http://localhost:8063/
Go to http://localhost:8063/search/

The search inputs should work and look like the designs

https://www.figma.com/design/Eux3guSenAFVvNHGi1Y9Wm/MIT-Design-System?node-id=3375-49855&m=dev
